### DISCUSSION TITLE
Specifying authorized symbols when generating password with regeneratePassword()

### DISCUSSION THREAD

**Discussion by arsounet** [rawilk/filament-password-input#5] [Q&A]
Can you tell me how you can limit authorized symbols  when generating password with regeneratePassword()?

**Comment by rawilk** [comment]
There's currently not a way to do that unless you specify a custom callback for re-generating a password, like this:

```php
Password::generatePasswordUsing(function ($state) {
    // Generate the password how you need to here.
});
```
